thrift.js

```diff
@@ -202,6 +310,19 @@ Thrift.prototype.link = function link() {
     var self = this;
     var index;
 
+    if (self.linked) {
+        return self;
+    }
+    self.linked = true;
+
+    var moduleNames = Object.keys(self.modulesByName);
+    for (index = 0; index < moduleNames.length; index++) {
+        var thriftModule = self.modulesByName[moduleNames[index]];
+        thriftModule.link(self);
+        self.modulesByName[moduleNames[index]] = thriftModule;
+        self[moduleNames[index]] = thriftModule;
```

Comment:
Let’s not add modules to the thrift object yet. We can use `thrift.modules.LocalModuleName` for now. We don’t have anything in place that would prevent a name collision with Thrift.prototype methods. We probably should fall back to only exposing services on the Thrift object proper since they have PascalCase names. We should also enforce that.